### ISSUE TITLE
CSP in <meta> is not analyzed when sent together with CSP in header

### ISSUE THREAD

**Issue by AntoniRoszak** [mozilla/http-observatory#489]
Steps:
1. Scan page: https://shop.rockwool.com

Observation:
- Page scores 120 with most CSP directives listed as "none".

Expectation:
- Page should score 110, as there are more directives in <meta> tag. They are analyzed by code, but discarded.

Problem appeared in commit a422b3aee91f34535990ffa7ba3aa5256dfb83da - when I check out master before this commit, the combined policy is analyzed properly.

CSP header data:
`upgrade-insecure-requests; frame-ancestors 'self'`

**Comment by mirunacurtean** [comment]
This appears to be fixed in 2023. 
![image](https://github.com/mozilla/http-observatory/assets/53306802/255a972e-5105-4f8b-a70f-f09d76948156)

**Comment by rw-AntoniRoszak** [comment]
Yes, but this is because in this case, CSP was moved to HTTP header. So the problem is no longer visible on the listed page.
Unfortunately I don't know any other page that could be used for problem reproduction.
